src/CFamily.UnitTests/Rules/DynamicCFamilyRulesConfigTests.cs

```diff
@@ -73,11 +74,14 @@ public void NullOrEmptyRulesSettings_DefaultsUsed()
             // Arrange
             var defaultConfig = new DummyCFamilyRulesConfig("123")
                 .AddRule("rule1", IssueSeverity.Blocker, isActive: true,
-                    parameters: new Dictionary<string, string> { { "p1", "v1" } })
+                    parameters: new Dictionary<string, string> { { "p1", "v1" } },
```

Comment:
Adding a separate test does not make sense for this. I have added here just because there is severity stuff here and asserts the rulesmetadata. More meaningful tests will be added during the translator task.